graylog2-server/src/main/java/org/graylog2/rest/resources/system/indexer/IndicesResource.java

```diff
@@ -173,6 +174,16 @@ public boolean apply(String indexName) {
         return ClosedIndices.create(reopenedIndices, reopenedIndices.size());
     }
 
+    @GET
+    @Timed
+    @Path("/all")
```

Comment:
I'm not too opinionated about it, but would it make sense to host this resource at "/system/indexer/indices" (i. e. where the `open()` method is currently placed)?